distribution/common/src/main/resources/etc/log4j2.config.xml

```diff
@@ -0,0 +1,150 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser General Public License as published by the Free Software Foundation, either
+ * version 3 of the License, or any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ *
+ **/
+-->
+<Configuration status="WARN">
+    <Properties>
+        <Property name="log-pattern">%-5.5p | %-16.16t | %-32.32c{1} | %-32.32C %4L | %X{bundle.id}
+            - %X{bundle.name} - %X{bundle.version} | %m%n
+        </Property>
+    </Properties>
+
+    <Appenders>
+        <PaxOsgi name="osgi-platformLogging"
+                 filter="org.codice.ddf.platform.logging.LoggingService"/>
+
+        <PaxOsgi name="osgi-all" filter="*"/>
+
+        <Syslog name="syslog" facility="AUTH" host="localhost" protocol="UDP" port="514"/>
+
+        <Console name="stdout" target="SYSTEM_OUT">
+            <PatternLayout pattern="%d{ABSOLUTE} | ${log-pattern}"/>
+        </Console>
+
+        <RollingFile name="out" fileName="${sys:karaf.data}/log/alliance.log"
```

Comment:
@shaundmorris, ddf changes to these log configuration files will not break Alliance, but the corresponding changes will need to be made in Alliance and any downstream projects.